cmd/liqoctl/cmd/peer.go

```diff
@@ -95,22 +105,31 @@ or
 `
 
 func newPeerCommand(ctx context.Context, f *factory.Factory) *cobra.Command {
+	options := &peer.Options{Factory: f}
 	cmd := &cobra.Command{
-		Use:   "peer",
-		Short: "Enable a peering towards a remote cluster",
-		Long:  liqoctlPeerLongHelp,
-		Args:  cobra.NoArgs,
+		Use:               "peer",
+		Short:             "Enable a peering towards a remote cluster",
+		Long:              liqoctlPeerLongHelp,
```

Comment:
```suggestion
		Long:              WithTemplate(liqoctlPeerLongHelp),
```

Otherwise the commands name does not get rendered.